functions/src/import/config.ts

```diff
@@ -55,10 +55,10 @@ const validateInput = (
   regex: RegExp,
   sizeLimit: number
 ) => {
-  if (!value || value === '' || value.trim() === '') {
+  if (!value || value === '' || !value.trim()) {
```

Comment:
Why was this not good ? `value.trim() === ''` 

To avoid a `trim is not a function` error we should check if the value is a string: 

```suggestion
  if (!value || typeof value !== 'string' || value === '' || !value.trim()) {
```

Reply:
I thought that was what typescript was for because is defined in the parameter as:
```
  value: string,
```

Reply:
In this case it can be even more simplified: 
```
if (value.trim() !== '')
```

But this function is checking the user inputs? In which case typescript wont help as it does not compile to a javascript with type checking. It is only for development purposes